tests/models/test_hparams.py

```diff
@@ -85,13 +85,24 @@ def __init__(self, *args, my_loss=torch.nn.CrossEntropyLoss(), **kwargs):
         self.auto_collect_arguments()
 
 
+class DictConfSubClassEvalModel(SubClassEvalModel):
+    def __init__(self, *args, dict_conf=DictConfig(dict(my_param='something')), **kwargs):
```

Comment:
Use OmegaConf.create() to initialize the config:

```suggestion
    def __init__(self, *args, dict_conf=OmegaConf.create({'my_param': 'something'}), **kwargs):
```

The DictConfig constructor is not a public API and should not be used normally by client code.